src/main/java/com/amihaiemil/docker/SystemDiskSpaceInfo.java

```diff
@@ -0,0 +1,60 @@
+package com.amihaiemil.docker;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+/**
+ * Docker disk space usage information.
+ * @author Boris Kuzmic (boris.kuzmic@gmail.com)
+ * @since 0.0.6
+ */
+public final class SystemDiskSpaceInfo implements DiskSpaceInfo {
```

Comment:
@bkuzmic Can you make this object polymorphic as JsonObject? See how [RtContainer](https://github.com/amihaiemil/docker-java-api/blob/master/src/main/java/com/amihaiemil/docker/RtContainer.java#L49) extends ``JsonResource``. 

Also, ``DiskSpaceInfo`` should extend the ``javax.json.JsonObject`` Interface :) 

The most important advantage is that the user will have access to the full representation of the json response, not only the convenience methods we are displaying in our interface.